Example/openxcframework.xcodeproj/project.pbxproj

```diff
@@ -277,6 +277,7 @@
 					};
```

Comment:
These changes were already approved and merged in another pull request. It can be removed from this one.

Reply:
These are not code change it is project setting change for the certificate code signing change .I cant change this settings it will create conflict during merge in master and ith is difficult to fix the settings things. So it wont affect any thing.

Reply:
And after merging previous one this branch is updated with that changes so no need to remove.